client/pages/AssessmentHome/FraHome/Collaborators/Collaborators.tsx

```diff
@@ -1,20 +1,41 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 
 import { Users } from '@meta/user'
 
+import { useAppDispatch } from '@client/store'
+import { useAssessment, useCycle } from '@client/store/assessment'
 import { useUser } from '@client/store/user'
-import { useUsers } from '@client/store/userManagement/hooks'
+import { UserManagementActions } from '@client/store/userManagement'
+import { useUsers, useUserToEdit } from '@client/store/userManagement/hooks'
 import { useCountryIso } from '@client/hooks'
+import EditUserForm from '@client/components/EditUserForm'
 import InviteUserForm from '@client/components/InviteUserForm'
 import UserList from '@client/components/UserList'
 
 const Collaborators: React.FC = () => {
-  const countryIso = useCountryIso()
+  const dispatch = useAppDispatch()
 
+  const countryIso = useCountryIso()
+  const assessment = useAssessment()
+  const cycle = useCycle()
   const user = useUser()
+
+  const userToEdit = useUserToEdit()
   const users = useUsers()
 
-  return (
+  useEffect(() => {
+    dispatch(UserManagementActions.setUserToEdit(null))
+  }, [])
+
+  useEffect(() => {
+    dispatch(
+      UserManagementActions.getUsers({ countryIso, assessmentName: assessment.props.name, cycleName: cycle.name })
+    )
+  }, [countryIso])
```

Comment:
wasn't this somewhere else ?

Reply:
yes, it was into FraHome. I removed it from FraHome and keep it here in Collaborators